src/main/java/de/retest/web/WebDataFilter.java

```diff
@@ -0,0 +1,28 @@
+package de.retest.web;
+
+import java.util.Arrays;
+import java.util.HashSet;
+import java.util.Set;
+
+// TODO This should be part of recheck and somehow use RecheckAdapter#convert to filter the result.
+public class WebDataFilter {
+
+	/**
+	 * Tags that will not be ignored even if they are not shown.
+	 */
+	private static final Set<String> specialTags = new HashSet<>( Arrays.asList( "head", "meta", "option", "title" ) );
+
+	public static boolean ignore( final WebData webData ) {
```

Comment:
Now we can easily create unit tests for this. What do you think (especially when it comes to naming)?

Reply:
Naming: shouldIgnore (Prädikat) vs. Ignore (Anweisung).